openedx/core/djangoapps/programs/utils.py

```diff
@@ -127,6 +127,34 @@ def engaged_programs(self):
 
         return programs
 
+    def does_course_still_qualify_as_in_progress(self, now, enrolled_run_modes, course):
```

Comment:
Please add a docstring explaining what this method does, what arguments it takes, and what it returns. See other methods for examples.